api/types/firewall_profiles.go

```diff
@@ -16,3 +23,44 @@ type Rule struct {
 	MaxPort  int    `json:"max_port" header:"MAX_PORT"`
 	CidrIP   string `json:"source" header:"SOURCE"`
 }
+
+// ConvertFlagParamsToRules converts received input rules parameters into a Firewall Profile rules array
+func (fp *FirewallProfile) ConvertFlagParamsToRules(rulesIn string) error {
+	compRegEx := regexp.MustCompile(`(?P<ip_protocol>\w{3})\/(?P<min_port>\d+)(?:-(?P<max_port>\d+)?)?:(?P<source>[a-zA-Z0-9.\/]+)`)
```

Comment:
It probably should be a global variable  in order to avoid compiling the regexp every time the function is called